backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc

```diff
@@ -11,4 +11,9 @@ You restore application backups by creating a xref:../../../backup_and_restore/a
 You can create xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers, before the application container starts, or in the application container itself.
 
 include::modules/oadp-creating-restore-cr.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+link:https://github.com/openshift/oadp-operator/blob/master/docs/scripts/dc-restic-post-restore.sh[Post-restore cleanup script]
```

Comment:
@kalexand-rh do we need additional sign-offs for links to Github scripts?

Reply:
Yes. Someone from support/PX needs to sign off on it, and I'm not sure if it's @sferich888 or someone else for OAPD.

Reply:
@sbeskin-redhat, you need someone from support to explicitly approve this github link or to remove it.

Reply:
@kalexand-rh 
I cannot remove this link - it leads to essential information. Who can approve it?

Reply:
@sbeskin-redhat It might be better to copy the information into the docs repo directly. Especially pointing to the master dev branch as it is now, that content could change in the future in a way that's incompatible with the current OADP release.

Reply:
@kalexand-rh 
In view of Scott's comment, I think I'll create a reference module with the required info and include it in the master doc instead of the link.

Reply:
+1 to Scott's idea. Has that content already been reviewed by QE? If not, it'll need to be reviewed before we can merge.

Reply:
@kalexand-rh 
Do I understand correctly that it is impossible to have an xref to a standalone module?

Reply:
@kalexand-rh 
Yes, that's what I suspected. Please take a look now. It has already been reviewed by QE.